common/src/main/java/com/wynntils/core/components/Managers.java

```diff
@@ -43,12 +43,13 @@ public final class Managers {
     public static final TranslationManager Translation = new TranslationManager();
 
     // Managers with dependencies, ordered alphabetically as far as possible
-    public static final ConfigManager Config = new ConfigManager(ConfigUpfixer, Json);
```

Comment:
Not that it matters much, but as the comment says I have tried to keep the managers sorted alphabetically, if possible. In this case, I think StorageManager could stay below SplashManager, since no dependency change require a shuffle.

Reply:
I have kind of wanted config and storage to be as close as possible. But I can change it

Reply:
Ok, keep it. It might that I merge then later on anyway.